### PR TITLE
⭐ Add example code for Stable Diffusion(1.5)

### PR DESCRIPTION
**Pull request by kadirnar**
There is no documentation for Stable Diffusion(1.5). I have added an example code for this. Is there anything else you would like me to add?

### CODE REVIEW

**Comment by kadirnar** [comment]
How do I solve this error?
```bash
pip install pyright
rye run pyright
```

Error Message:
```
error: invalid or unknown script 'pyright' 
```

File: src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py [hunk 1]
```diff
@@ -29,6 +29,41 @@ class StableDiffusion_1(LatentDiffusionModel):
...
+
+    from refiners.fluxion.utils import manual_seed, no_grad
+    from refiners.foundationals.latent_diffusion.stable_diffusion_1 import StableDiffusion_1
+
+    # Load SD
+    sd15 = StableDiffusion_1(device="cuda", dtype=torch.float16)
+
+    sd15.clip_text_encoder.load_from_safetensors("realistic_vision.v5_1.sd1_5.text_encoder.safetensors")
+    sd15.unet.load_from_safetensors("realistic_vision.v5_1.sd1_5.unet.safetensors")
+    sd15.lda.load_from_safetensors("realistic_vision.v5_1.sd1_5.autoencoder.safetensors")
```
Comment (catwell) [inline src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py:45]:
```suggestion
    sd15.clip_text_encoder.load_from_safetensors("sd1_5.text_encoder.safetensors")
    sd15.unet.load_from_safetensors("sd1_5.unet.safetensors")
    sd15.lda.load_from_safetensors("sd1_5.autoencoder.safetensors")
```

Comment (catwell) [inline src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py:45]:
```suggestion
    sd15.clip_text_encoder.load_from_safetensors("sd1_5.text_encoder.safetensors")
    sd15.unet.load_from_safetensors("sd1_5.unet.safetensors")
    sd15.lda.load_from_safetensors("sd1_5.autoencoder.safetensors")
```

File: src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py [hunk 2]
```diff
@@ -29,6 +29,41 @@ class StableDiffusion_1(LatentDiffusionModel):
...
+
+    # Hyperparameters
+    prompt = "RAW photo, face portrait photo of beautiful 26 y.o woman, cute face, wearing black dress, happy face, hard shadows, cinematic shot, dramatic lighting"
```
Comment (catwell) [inline src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py:48]:
```suggestion
    prompt = "a cute cat, best quality, high quality"
    negative_prompt="monochrome, lowres, bad anatomy, worst quality, low quality"
```

File: src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py [hunk 3]
```diff
@@ -29,6 +29,41 @@ class StableDiffusion_1(LatentDiffusionModel):
...
+    seed = 42
+
+    sd15.set_inference_steps(50)
+
+    with no_grad():  # Disable gradient calculation for memory-efficient inference
+        clip_text_embedding = sd15.compute_clip_text_embedding(text=prompt + ", best quality, high quality", negative_text="monochrome, lowres, bad anatomy, worst quality, low quality")
```
Comment (catwell) [inline src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py:54]:
```suggestion
        clip_text_embedding = sd15.compute_clip_text_embedding(text=prompt, negative_text=negative_prompt)
```

File: src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py [hunk 4]
```diff
@@ -29,6 +29,41 @@ class StableDiffusion_1(LatentDiffusionModel):
...
+        manual_seed(seed)
+
+        # SDXL typically generates 1024x1024, here we use a higher resolution.
```
Comment (catwell) [inline src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py:57]:
```suggestion
```

File: src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py (hunk 2 above)
Comment (catwell) [reply src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py:48]: (let's not put too specific prompts in examples)

File: src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py (hunk 4 above)
Comment (catwell) [reply src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py:57]: (This comes from [this guide](https://refine.rs/guides/adapting_sdxl/) but it does not concern this example.)

File: src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py (hunk 1 above)
Comment (catwell) [reply src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py:45]: (let's not use specific models in examples)

**Review by catwell — CHANGES_REQUESTED** [review]

**Comment by kadirnar** [comment]
I ran this code and there was no change. Can you review it again?
```bash
rye run pyright
```

**Review by catwell — APPROVED** [review]

**Comment by catwell** [comment]
@kadirnar Can you squash all the commits into the first one?

I'd do it but with the way this repository is configured I cannot for external PRs like this one (because the main branch is protected and squash in the Git UI is disabled).

One way to do it is with interactive rebase, i.e. run `git rebase -i HEAD~10` and turn the last 5 commits into fixups like in the screenshot below, then force push to the branch.

Screenshot from 2024-08-31 11-25-25

**Comment by kadirnar** [comment]
I rolled back the previous commits and fixed them manually. Can you check?

**Comment by catwell** [comment]
It's merged, thanks!

**Pull request merged**